### DISCUSSION TITLE
Very simple rule does not apply ?

### DISCUSSION THREAD

**Discussion by rackilucky** [arendst/Tasmota#14018] [Q&A] [answered]
Dear all,

I have cleared my Rule1 with Rule1 '' and created a new one, that should immediately fire as soon as the analog value is above 200

But what it does, is fire once and thats it ... nothing else

This is the latest output I get:

18:48:53.121 MQT: esp/esp013/RESULT = {"Rule1":{"State":"ON","Once":"ON","StopOnError":"OFF","Length":61,"Free":450,"Rules":"ON analog#a0>200 DO publish esp/%topic%/SENSOR %value% ENDON "}}
18:48:53.688 RUL: ANALOG#A0>200 performs "publish esp/esp013/SENSOR 239"
18:48:53.694 MQT: esp/esp013/SENSOR = 239

**Comment by barbudor** [comment, marked as answer]
Because `"Once":"ON"`
It will not fire again until Analog goes back below 200 and again above 200.
This is because you previously used `Rule1 5` to enable once mode.
Use `Rule1 4` to disable it

**Reply by rackilucky** [reply]
Yes, it seems that you are right !
You are a very nice guy - thank to 1000 times for your quick answer !
You made my evening succesful and happy !
